.travis.yml

```diff
@@ -0,0 +1,53 @@
+sudo: false
+dist: trusty
+language: php
+notifications:
+  email:
+    on_success: never
+    on_failure: never
+  slack:
+    rooms:
+      - secure: DOpF7wER9onzQOw2nAU0jgsmfHmS7UhbMnvvzcMYD+wzWTVT6ccTG9yc93jzUZFlUeVTuDAMa/ws04qwc/an4bIU5Baxg+Hajj9iMMO3Rwft4jgGEgqUgSGmxYt/QYyv5ujV/3H2Yb/SLqa9d9ryGob3Kh263WG6/7jTuIzz5T8=
+    on_success: change
+    on_failure: always
+branches:
+  only:
+  - master
```

Comment:
I don't understand this. Why only `master`? And how is it working on this branch if we're configured to only run on `master`?